### PR TITLE
Add NEUTRONRCF435MINI

### PR DESCRIPTION
**Pull request by ASDosjani**
To make all uarts and baro work firmware changes required. I made this target from atbetaflight repo and a dump from the aio.

I need help to set baro feature defaultly on.

### CODE REVIEW

File: configs/NEUTRONRCF435MINI/config.h [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+#define DEFAULT_BLACKBOX_DEVICE BLACKBOX_DEVICE_FLASH
+#define DEFAULT_DSHOT_BURST DSHOT_DMAR_AUTO
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define BEEPER_INVERTED
+#define SYSTEM_HSE_MHZ          8
+#define USE_SPI_GYRO
+#define GYRO_1_SPI_INSTANCE     SPI1
+#define GYRO_1_ALIGN CW0_DEG
```
Comment (haslinghuis) [inline configs/NEUTRONRCF435MINI/config.h:102]: Can be removed - as it defaults to CW0_DEG

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/NEUTRONRCF435MINI/config.h [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+#define UART3_TX_PIN            PB11
+#define UART5_TX_PIN            PB9
+#define UART7_TX_PIN            PB4
+#define UART1_RX_PIN            PA10
+#define UART2_RX_PIN            PB0
+#define UART3_RX_PIN            PB10
+#define UART5_RX_PIN            PB8
+#define UART7_RX_PIN            PB3
+#define I2C2_SCL                PH2
+#define I2C2_SDA                PH3
```
Comment (haslinghuis) [inline configs/NEUTRONRCF435MINI/config.h:68]:
```suggestion
#define I2C2_SCL_PIN            PH2
#define I2C2_SDA_PIN            PH3
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/NEUTRONRCF435MINI/config.h (hunk 1 above)
Comment (ASDosjani) [reply configs/NEUTRONRCF435MINI/config.h:102]: Can I delete GYRO_1_ALIGN_YAW as well?

Comment (haslinghuis) [reply configs/NEUTRONRCF435MINI/config.h:102]: Yes zero should be the default here:

    "gyro_1_align_yaw", VAR_INT16  | HARDWARE_VALUE, .config.minmax = { -3600, 3600 }, PG_GYRO_DEVICE_CONFIG, PG_ARRAY_ELEMENT_OFFSET(gyroDeviceConfig_t, 0, customAlignment.yaw) },

File: configs/NEUTRONRCF435MINI/config.h [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+#define FC_TARGET_MCU        AT32F435G
+
+#define BOARD_NAME           NEUTRONRCF435MINI
+#define MANUFACTURER_ID      NERC
+
+#define USE_ACC
+#define USE_GYRO
+#define USE_ACCGYRO_BMI270
+
+#define USE_ACC
```
Comment (haslinghuis) [inline configs/NEUTRONRCF435MINI/config.h:33]: duplicated, can be removed

**Review by haslinghuis — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

File: configs/NEUTRONRCF435MINI/config.h [hunk 4]
```diff
@@ -104,3 +104,8 @@
 #define LED0_INVERTED
 #define LED1_INVERTED
 #define DEFAULT_CURRENT_METER_SCALE 100
+
+#define DEFAULT_FEATURES                (FEATURE_OSD | FEATURE_TELEMETRY | FEATURE_SOFTSERIAL |FEATURE_LED_STRIP )
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
+#define SERIALRX_PROVIDER       SERIALRX_CRSF
```
Comment (haslinghuis) [inline configs/NEUTRONRCF435MINI/config.h:110]: defined by [cloud] build options.

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**